### ISSUE TITLE
Form control element <textarea> has no associated label - causes accessibility issue

### ISSUE THREAD

**Issue by manjunbm** [codemirror/CodeMirror#6832]
The textarea coming as part of Codemirror is not associated with any label, this causes the accessibility issue when you can scan a component built using Codemirror with a Accessibility checker tool. 

_When we scan component built with Codemirror we are hitting below error:_
**Error:**
"Form control element <textarea> has no associated label"

**What it means:**
Each form control must be associated with a label.

**Why is it required:**
Associating a meaningful label with every UI control allows the browser and assistive technology to expose and announce the control to a user. Associating a visible label also provides a larger clickable area.

**Element location :**
<textarea style="position: absolute; bottom: -1em; padding: 0px; width: 1000px; height: 1em; min-height: 1em; outline: none;" tabindex="0" spellcheck="false" autocapitalize="off" autocorrect="off">

**What to do**
If the control is a labelable element, add a for attribute to a visible label with the id of this control (e.g. <label for="controlID">);
Or, if the control is a labelable element, put the control inside a visible <label> element;
Or, add an aria-labelledby attribute to the control. It must point to visible text on the page that is meaningful as a label;
Or, if the design cannot have a visible label, provide a label using the aria-label attribute (e.g. aria-label="Activities");
Or, if the design cannot have a visible label, use the title attribute to provide a label.

Kindly get this fixed as not having label to _textarea_ causes accessibility issue
